frontend/src/modules/member/components/list/member-list-table.vue

```diff
@@ -133,7 +133,8 @@
                 <template #default="scope">
                   <app-member-organizations
                     :member="scope.row"
-                    :show-title="false"
+                    :show-title="true"
+                    orientation="horizontal"
```

Comment:
Not sure if intended but in staging the UI looks a bit broken (first image). Shouldn't the orientation be vertical maybe? (second image)
Just checking 🙏 
![Screenshot 2023-01-17 at 14 20 43](https://user-images.githubusercontent.com/20134207/212923251-ef5b6530-6a03-45dd-9fd9-725b28cdece2.png)

<img width="598" alt="Screenshot 2023-01-17 at 14 20 37" src="https://user-images.githubusercontent.com/20134207/212923301-c4d1ebe1-395e-4260-9f8b-79973a4c8b67.png">

Reply:
Oh you're right, thanks for flagging this
Will fix it!